lib/solveEquation/stepThrough.js

```diff
@@ -68,6 +70,52 @@ function stepThrough(leftNode, rightNode, comparator, debug=false) {
     }
 
     try {
+      if (equation.leftNode.op === '*'
+          && (Node.Type.isConstant(equation.rightNode)
+              && parseFloat(equation.rightNode.value) === 0)) {
+        // Gets all factors that aren't a constant
+        const factors = equation.leftNode.args.filter(arg => !Node.Type.isConstant(arg));
+
+        const solutions = [];
+        let symbol;
+
+        // For each factor, solve the equation factor = 0 for the symbol
+        for (var f in factors) {
+          const leftNode = Node.Type.isParenthesis(factors[f])
+                ? factors[f].content
+                : factors[f];
+
+          const rightNode = Node.Creator.constant(0);
+
+          const allSteps = stepThrough(leftNode, rightNode, '=');
+
+          if (allSteps.length > 0) {
+            const lastStep = stepThrough(leftNode, rightNode, '=').slice(-1)[0];
+
+            symbol = lastStep.newEquation.leftNode;
```

Comment:
@aliang8 has this been addressed? if it's something we don't support is it in a TODO in the test cases?